build.bash

```diff
@@ -0,0 +1,80 @@
+# Copyright 2024 Adevinta
+
+if [[ -z $LAVA_VERSION ]]; then
+	echo 'error: missing env var LAVA_VERSION' >&2
+	exit 2
+fi
+
+check_status (){
+	local status=$1
+	local error_msg=$2
+	if [[ $status -ne 0 ]]; then
+		echo $error_msg
+		exit $status
+	fi
+}
+
+fix_headers (){
+	local file=$1
+	sed -i "s/^#/##/" ${file}
+	check_status $? "error: could not fix headers in file ${file}"
+
+}
+
+add_main_header(){
+	local file=$1
+	local header=$2
+	sed -i "1s/^/# ${header}\n/" ${file}
+	check_status $? "error: could not add main header ${header} to file ${file}"
+}
+
+GH_DOWNLOAD_URL_FMT='https://github.com/adevinta/lava/releases/download/%s/lava_linux_amd64.tar.gz'
+GH_DOWNLOAD_URL_LATEST='https://github.com/adevinta/lava/releases/latest/download/lava_linux_amd64.tar.gz'
+
+install_lava() {
+	local version=$1
+
+	local url
+	if [[ $version == 'latest' ]]; then
+		url=$GH_DOWNLOAD_URL_LATEST
+	else
+		url=$(printf "${GH_DOWNLOAD_URL_FMT}" "${version}")
+	fi
+
+	# Make sure the install directory exists.
+	install_dir="/tmp/bin"
+	mkdir -p "${install_dir}"
```

Comment:
I would move this out of this function and pass the install dir as a parameter of the function before calling it. Also I would generate a directory using `mktemp`.